Promact.OAuth.Client/src/Promact.OAuth.Client/Promact.OAuth.Client.nuspec

```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>Promact.OAuth.Client</id>
     <version>$version$</version>
-    <title>Promact.OAuth.Client</title>
+    <title>Promact-OAuth-SDK</title>
```

Comment:
When you will add the environment variable of travis.  I will updated it and then will ask you to accept the request.

I changed the name for publishing it in nuget from my account